src/components/KeyboardAvoidingView/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2015-present, Nicolas Gallagher.
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * @providesModule KeyboardAvoidingView
+ * @flow
+ */
+
+import View from '../View';
+export default View;
```

Comment:
Please make this an accurate stub of the view: https://facebook.github.io/react-native/docs/keyboardavoidingview.html